omnibus/config/projects/chef.rb

```diff
@@ -51,15 +51,27 @@
 # Chef Release version pinning
 override :chef, version: "local_source"
 override :ohai, version: "master"
+override :"rb-readline", version: "v0.5.3"
```

Comment:
if there's no reason to override it, then you're just creating meaningless technical debt